airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSourceOperations.java

```diff
@@ -107,7 +107,7 @@ public void setJsonField(final ResultSet resultSet, final int colIndex, final Ob
    */
   @Override
   protected void putBoolean(final ObjectNode node, final String columnName, final ResultSet resultSet, final int index) throws SQLException {
-    node.put(columnName, resultSet.getInt(index) == 1);
+    node.put(columnName, resultSet.getInt(index) > 0);
```

Comment:
This is not related to debezium version upgrade, I found a bug while testing in Full refresh mode so went ahead and fixed it